src/lib/skilling/skills/prayer.ts

```diff
@@ -0,0 +1,135 @@
+import { Bone, SkillsEnum } from '../types';
+import itemID from '../../util/itemID';
+import { Emoji } from '../../constants';
+
+const bones: Bone[] = [
+	{
+		name: 'bones',
```

Comment:
why all bones arent cased properly? rs items always have the first letter capitalized